c2corg_ui/__init__.py

```diff
@@ -28,6 +28,8 @@ def main(global_config, **settings):
     config.add_route('waypoints_view', '/waypoints/{id}/{lang}')
     config.add_route('waypoints_index_default', '/waypoints')
     config.add_route('waypoints_index', '/waypoints/list*filters')
```

Comment:
Right! At first I thought I would use a ``key/value`` syntax for URLs but it seems it is not possible with ``ngeo.Location`` and anyway I'm not sure this syntax would be useful since the page heavily relies on AJAX requests and thus can't really be cached. So I decided not to spend time using this special syntax.
Then this route is indeed no longer needed. I will remove it. Thanks for noticing :)